src/main/java/dev/t3d/happraisal/HappraisalApplication.java

```diff
@@ -0,0 +1,13 @@
+package dev.t3d.happraisal;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class HappraisalApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(HappraisalApplication.class, args);
```

Comment:
When I start the application I get a org.h2.jdbc.JdbcSQLDataException: Value too long for column "ID BINARY": "CAST(X'4b6afd254a23436faa7eb9cbad0cc95b' AS BINARY(16)) (16)"; SQL statement: